### ISSUE TITLE
Remote target="_blank" markup

### ISSUE THREAD

**Issue by mgottholsen** [civicdata/civicdata.github.io#34]
See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

**Comment by cheapwebmonkey** [comment]
Good catch! Please ref https://github.com/asciidoctor/asciidoctor/issues/2071 for more detail on the effects on analytics of using the flavor `rel="noopener noreferrer"` vs  `rel="noopener"`

**Comment by 73VW** [comment]
I will work on this if you accept.

Do you want to add `rel="noopener noreferrer"` or not taking care of old browsers and use `rel="noopener"` ?

Cheers.

**Comment by cheapwebmonkey** [comment]
> rel="noopener"

rel="noopener" is awesome to use :)
